site/testsuites/testsuite-kotlin/src/test/kotlin/docs/utils/MockExchangesApiProvider.kt

```diff
@@ -0,0 +1,122 @@
+package website.tbd.developer.site.docs.utils
+
+import website.tbd.developer.site.docs.utils.MockDataProvider
+import tbdex.sdk.httpserver.models.*
+import tbdex.sdk.protocol.models.*
+
+open class MockExchangesApiProviderTest: ExchangesApi {
```

Comment:
```suggestion
open class MockExchangesApiProvider: ExchangesApi {
```

classname should match filename